spec/index.md

```diff
@@ -439,6 +439,16 @@ The state tree contains information about the topology of the Internet Computer.
         principal = bytes .size (0..29)
         tagged<t> = #6.55799(t) ; the CBOR tag
 
+-   `/subnet/<subnet_id>/metrics` (blob)
+
+     A collection of subnet-wide metrics related to this subnet's current resource usage and/or performance. The metrics are a CBOR map with the following fields:
+     
+     - `num_canisters` (`nat`): The number of canisters on this subnet.
+     - `total_canister_state` (`nat`): The total size of the state in bytes taken by canisters on this subnet since this subnet was created.
+     - `total_consumed_cycles` (`map`): The total number of cycles consumed by canisters on this subnet since this subnet was created. It's a map of two values, a low part of type `nat` and a high part of type `opt nat`.
+     - `num_update_transactions` (`nat`): The total number of transactions processed on this subnet since this subnet was created.
```

Comment:
Not sure how transactions are counted, but consumed cycles are summed across all canisters (to which the cycles burned by deleted canisters are added). Currently, this is indeed "cycles consumed [...] since this subnet was created". But if we consider subnet splitting/canister migration, this no longer holds.

Maybe it makes sense to reword this (or maybe not). That being said, I don't have a suggestion for how to make this both generic and concise. Maybe "the sum of all-time burned cycles by all current and deleted canisters".

Reply:
@mraszyk any thoughts on this? I'm ok with "the sum of all-time burned cycles by all current and deleted canisters".

Re update_transactions: we keep accumulating them in a counter at the end of every round execution. So, it would include indeed all transactions since the subnet was created (even ones that were for canisters that were migrated away).

Reply:
I'm also fine with the formulation "the sum of all-time burned cycles by all current and deleted canisters".

Reply:
Ok, changed it to the proposed formulation.